_includes/apply_form.html

```diff
@@ -51,21 +56,22 @@ <h4 class="modal-title" id="myModalLabel4">{{ include.title }}</h4>
                         </div>
                         <div class ="row">
                             <div class="col-md-12 margin-bottom-20">
-                                <label>What does Software Craftmanship mean to you?</label>
+                                <label>What does Software Craftmanship mean to you? (*)</label>
                                 <textarea name="{{ include.target }}-craftmanship" class="form-control" rows="10" id="craftmanship"></textarea>
```

Comment:
Same as above:
On this case we are not using required as we are really not submitting the form, but calling a js method that does an Ajax call.